DESCRIPTION

```diff
@@ -29,6 +29,7 @@ Imports:
     shinyWidgets,
     styler
 Suggests:
+    grDevices,
```

Comment:
fixing previous mistake of not adding a package that is used in the tests

Reply:
actually this should be even in imports as one functionality depends on this!

Reply:
This is on main now:
https://github.com/insightsengineering/teal.widgets/blob/f75ce746b45253ebcc1dadd3a022d49b2b5f49e0/R/plot_with_settings.R#L406-L408